operator/v1/types_etcd.go

```diff
@@ -43,6 +43,14 @@ type EtcdSpec struct {
 	// +openshift:enable:FeatureGate=HardwareSpeed
 	// +optional
 	HardwareSpeed ControlPlaneHardwareSpeed `json:"controlPlaneHardwareSpeed"`
+
+	// BackendQuotaGiB sets the etcd backend storage size limit in gigabytes. Defaults to 8GiB.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=8
+	// +kubebuilder:validation:Maximum=64
+	// +kubebuilder:validation:XValidation:rule="self >= oldSelf",message="can't decrease database size"
+	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
```

Comment:
I owe docs (not yet written), but just this week we've switched this to using FeatureGates, so you'll need a FeatureGate associated and regenerate.  See example here: https://github.com/openshift/api/blob/master/example/v1/types_stable.go#L40-L42

Reply:
@deads2k so right now i made the annotations 

```
// +openshift:enable:FeatureGate=BackendQuotaGiB
```

what about `FeatureGate` ? 

cc @tjungblu @soltysh @dusk125